### PR TITLE
Fixed the billboard alignedaxis issue.

### LINKED ISSUE #2487: Billboard rotation accuracy issue
Paste this into Sandcastle, switch to 2D or CV and then pan/rotate around the map.  The arrow points down by default, so the below should make it always point south.  However, when you pan around the map, the arrow rotates slightly.  It happens in 3D too, but is more noticeable in other modes.  @bagnell thinks it's a precision issue in the shader.

``` javascript
var viewer = new Cesium.Viewer('cesiumContainer');
viewer.entities.add({
    position : Cesium.Cartesian3.fromDegrees(0, 0, 0),
    billboard : {
        image : 'http://maps.google.com/mapfiles/kml/shapes/arrow.png',
        alignedAxis : Cesium.Cartesian3.UNIT_Z
    }
});
```

### PR DESCRIPTION
**Pull request by klingerj**
Fixes #2487

### CODE REVIEW

File: Source/Shaders/BillboardCollectionVS.glsl [hunk 1]
```diff
@@ -256,6 +256,7 @@ void main()
 
     vec4 positionWC = computePositionWindowCoordinates(positionEC, imageSize, scale, direction, origin, translate, pixelOffset, alignedAxis, validAlignedAxis, rotation, sizeInMeters);
     gl_Position = czm_viewportOrthographic * vec4(positionWC.xy, -positionWC.z, 1.0);
+    //gl_Position = czm_modelViewProjection * vec4(positionWC.xy, -positionWC.z, 1.0);
```
Comment (emackey) [inline Source/Shaders/BillboardCollectionVS.glsl]: Oops... extra comment slipped by.

File: Source/Shaders/BillboardCollectionVS.glsl [hunk 2]
```diff
@@ -53,9 +53,9 @@ vec4 computePositionWindowCoordinates(vec4 positionEC, vec2 imageSize, float sca
         {
             vec3 pos = positionEC.xyz + czm_encodedCameraPositionMCHigh + czm_encodedCameraPositionMCLow;
```
Comment (bagnell) [inline Source/Shaders/BillboardCollectionVS.glsl]: You can remove this and `normal` as well.

**Comment by bagnell** [comment]
@klingerj When you finish updates after a review, bump the PR so the reviewer knows to look again. Please  update any forum threads linked in the issue that this was fixed and will be in the next release.

**Comment by klingerj** [comment]
Got it, thanks for letting me know @bagnell.

**Comment by pjcozzi** [comment]
Fantastic, thanks again @klingerj!

**Pull request merged**